src/main/telemetry/telemetry.c

```diff
@@ -226,3 +232,90 @@ void telemetryProcess(uint32_t currentTime)
 #endif
 }
 #endif
+
+typedef struct sensorTableEntry_s {
+    sensor_e sensor;
+    uint32_t requiredFeatures;
+#ifdef SENSOR_NAMES
+    const char *name;
+#endif
+} sensorTableEntry_t;
+
+#ifdef SENSOR_NAMES
+#define SENSOR_ENTRY(a,b,c) a,b,c
+#else
+#define SENSOR_ENTRY(a,b,c) a,b
+#endif
+
+static const sensorTableEntry_t sensorTable[] = {
+    { SENSOR_ENTRY(ESC_SENSOR_CURRENT, FEATURE_ESC_SENSOR, "ESC_CURRENT") },
+    { SENSOR_ENTRY(ESC_SENSOR_VOLTAGE, FEATURE_ESC_SENSOR, "ESC_VOLTAGE") },
+    { SENSOR_ENTRY(ESC_SENSOR_RPM, FEATURE_ESC_SENSOR, "ESC_RPM") },
+    { SENSOR_ENTRY(ESC_SENSOR_TEMPERATURE, FEATURE_ESC_SENSOR, "ESC_TEMPERATURE") },
+};
+
+#define SENSOR_TABLE_ENTRY_COUNT (sizeof(sensorTable) / sizeof(sensorTableEntry_t))
+
+#define SENSOR_GET_FLAG(sensor) (1 << (sensor - 1))
+
+bool telemetryIsSensorEnabled(sensor_e sensor)
+{
+    return telemetryConfigMutable()->enabledSensors & SENSOR_GET_FLAG(sensor);
+}
+
+void telemetrySensorSet(sensor_e sensor, uint32_t *sensors)
```

Comment:
In C, destination is usually on left side - first argument.
With objects, object (`sensors`) comes first